src/Controller/PageAdminController.php

```diff
@@ -256,22 +268,18 @@ public function composeAction(Request $request): Response
      */
     public function composeContainerShowAction(Request $request): Response
     {
-        $blockAdmin = $this->container->get('sonata.page.admin.block');
-        \assert($blockAdmin instanceof BlockAdmin);
-
-        if (false === $blockAdmin->isGranted('LIST')) {
+        //TODO Can I use this directly from symfony controller?
+        if (false === $this->isGranted('LIST')) {
             throw new AccessDeniedException();
         }
 
         $id = $request->get($this->admin->getIdParameter());
-        $block = $blockAdmin->getObject($id);
+        $block = $this->admin->getObject($id);
```

Comment:
This admin is a pageAdmin, not a block, it will return a page, this change is not correct.